test/phantom-test.js

```diff
@@ -33,7 +33,6 @@ describe('phantom', function () {
       assert.equal(stdout.indexOf('# phantomjs:\n'
         + '1..1\n'
         + 'not ok 1 test fails\n'
-        + '  Error: Oh noes!\n'
```

Comment:
This change fails on travis and also for me locally. The other newline changes are fixing it for me as well.